catroid/src/androidTest/java/org/catrobat/catroid/test/ui/MultilingualTest.java

```diff
@@ -0,0 +1,186 @@
+
+	public MultilingualTest() {
+		super(MainMenuActivity.class);
+	}
+
+	private Solo solo;
+
+	@Override
+	protected void setUp() throws Exception {
+		solo = new Solo(getInstrumentation(), getActivity());
+	}
+
+	@Override
+	protected void tearDown() throws Exception {
+		solo.finishOpenedActivities();
+	}
+
+	@Override
+	protected void runTest() throws Throwable {
+		super.runTest();
+	}
+
+	public void testArabicText() throws Exception {
+
+		// make English as App_Language
+		solo.sendKey(Solo.MENU);
+		solo.clickInList(5);
+		solo.clickInList(1);
+		solo.assertCurrentActivity("Language", Multilingual.class);
+		solo.sleep(2000);
+		solo.scrollDown();
+		solo.clickInList(4);
+		solo.waitForActivity(MainMenuActivity.class);
+		solo.assertCurrentActivity("MainMenu", MainMenuActivity.class);
+		//
+		String s0 = getActivity().getString(R.string.app_name);
+		String s01 = getActivity().getString(R.string.programs);
+		// make Arabic as App_Language
+		solo.sendKey(Solo.MENU);
+		solo.clickInList(5);
+		solo.clickInList(1);
+		solo.assertCurrentActivity("Language", Multilingual.class);
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.clickInList(4);
+		solo.sleep(2000);
+		solo.waitForActivity(MainMenuActivity.class);
+		solo.assertCurrentActivity("MainMenu", MainMenuActivity.class);
+		String s = getActivity().getString(R.string.app_name);
+		assertEquals(s, "بوكت كوود");
+		assertNotSame(s0, "s");
+		String s1 = getActivity().getString(R.string.programs);
+		assertEquals(s1, "البرامج");
+		assertNotSame(s01, s1);
+	}
+
+	public void testArabic_LayoutDirection() throws Exception {
+		solo.sleep(5000);
+		int L = getActivity().getResources().getConfiguration().orientation;
+		int L1 = LayoutDirection.RTL;
+		assertEquals(L1, L);
+	}
+	public void testFarsiText() throws Exception {
+
+		// make English as App_Language
+		solo.sendKey(Solo.MENU);
+		solo.clickInList(5);
+		solo.clickInList(1);
+		solo.assertCurrentActivity("Language", Multilingual.class);
+		solo.sleep(2000);
+		solo.scrollDown();
+		solo.clickInList(4);
+		solo.waitForActivity(MainMenuActivity.class);
+		solo.assertCurrentActivity("MainMenu", MainMenuActivity.class);
+		//
+		String s0 = getActivity().getString(R.string.app_name);
+		String s01 = getActivity().getString(R.string.programs);
+		// make Farsi as App_Language
+		solo.sendKey(Solo.MENU);
+		solo.clickInList(5);
+		solo.clickInList(1);
+		solo.assertCurrentActivity("Language", Multilingual.class);
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.scrollDown();
+		solo.clickInList(6);
+		solo.sleep(2000);
+		solo.waitForActivity(MainMenuActivity.class);
+		solo.assertCurrentActivity("MainMenu", MainMenuActivity.class);
+		String s = getActivity().getString(R.string.app_name);
+		assertEquals(s, "پاکت کد");
```

Comment:
can you please put all those button names and expected strings into static final string variables with english names (e.g. BUTTON_NAME_PROGRAMS) at the top of this test class, so developers who cannot read these languages can understand what this test does